aeneas/src/vst/Parser.v3

```diff
@@ -729,6 +767,31 @@ component Parser {
 			return null;
 		}
 	}
+	def parseUnitDecimal(p: ParserState) -> VstUnitDecimal {
+		if (!Char.isDecimal(p.curByte)) {
+			p.error("invalid struct size format");
+			return null;
+		}
+		var np = NumberParser.new(p.input, p.curPos);
```

Comment:
As discussed, I don't think you want to use a `NumberParser` here, because it will accept floating point numbers.

Reply:
Got it! numberparser is now removed.